src/app/containers/Kvote/KvoteContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {useTranslation} from "react-i18next";
+import {apiRoutes} from "../../../utils/http/apiConfig";
+import useGet from "../../../utils/http/useGet";
+import navColors from "../../../styles/designSystemColors";
+
+import RammemeldingOverskrift from "../../components/RammemeldingOverskrift/RammemeldingOverskrift";
+import {LoadingIndicator} from "../../components/LoadingIndicator";
+import PanelMedDagerOgBeskrivelse from "../../components/PanelMedDagerOgBeskrivelse/PanelMedDagerOgBeskrivelse";
+import {CalendarIcon} from "../../components/icons";
+
+interface Props {
+    saksnummer: string;
+};
+
+export const KvoteContainer: React.FunctionComponent<Props> = ({saksnummer}) => {
+    const {t} = useTranslation();
+    const {data, error, loading} = useGet(`${apiRoutes().Kvote}?status=Aktiv&saksnummer=${saksnummer}`);
+    const errorStatus = error?.response?.status;
+    const år = new Date().getFullYear().toString();
+
+    return (
+        <>
+            <RammemeldingOverskrift>
+                <CalendarIcon />
+                {t('kvote.overskrift', {år: år})}
+            </RammemeldingOverskrift>
+
+            {loading && <LoadingIndicator/>}
+
+            {error && (<p>
+                {
+                    errorStatus === 404 ? (
+                        <>
+                            {t('sak.feil.404')}
+                            <code>{`${saksnummer}`}</code>
+                        </>
+                    ) : (
+                        t('sak.feil.ukjent')
+                    )
+                }
+            </p>)
+            }
```

Comment:
Format; `{`, `(` og `<p>` var på samme linje - da skal også `</p>`, `)` og `}` være på samme linje.
```suggestion
            </p>)}
```